### PR TITLE
Gogonimago test changes may 14

### PR DESCRIPTION
**Pull request by gogonimago**
# Description

I am testing VS Code pls ignore this PR

# Changes

- [ ] ...
- [ ] ...

### CODE REVIEW

**Comment by gogonimago** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by gogonimago** [comment]
recheck

**Comment by gogonimago** [comment]
Context: This was originally meant to be a test PR as I was uploading from VS Code from the first time, but every time I saved it committed changes to the same PR, so now this has become **the** authoritative PR for the entirety of my copy changes to the MEV Blocker docs.

File: docs/mevblocker/README.mdx [hunk 1]
```diff
@@ -1,43 +1,44 @@
...
+## MEV Blocker Offers Important Advantages to Users
 
-* **Faster Inclusion over other RPCs**: Instead of sending your transaction to the Ethereum public mempool with a low tip and waiting for it to be included, you send your transaction to MEV Blocker private pool which forwards your transactions directly to all major builders. This yields a faster inclusion rate as your transaction is already on the builders' radar and will likely be included in the current or next block available.
+* **MEV Protection**: Transactions sent to the public pool are subject to all sorts of MEV attacks including [frontrunning](https://blog.cow.fi/what-is-frontrunning-b2641ad7af21) and [sandwich attacks](https://blog.cow.fi/what-is-a-sandwich-attack-and-how-can-you-protect-yourself-b101c9a9b9b3). MEV bots are responsible for [millions of dollars of value lost each week](https://eigenphi.io/mev/ethereum/sandwich). By using MEV Blocker's private pool, all transactions are de-facto protected from harmful MEV.
```
Comment (cowmarketing) [inline docs/mevblocker/README.mdx]: to the public mempool

File: docs/mevblocker/README.mdx [hunk 2]
```diff
@@ -1,43 +1,44 @@
...
 
-* **Transparency**: MEV Blocker has a [Dune Dashboard](https://dune.com/cowprotocol/mev-blocker)
-where you can check all your transactions submitted through the MEV Blocker RPC. You can also follow your transactions in detail, see all searcher bids, and calculate the overall revenue you got via MEV Blocker rebates.
+* **Faster Inclusion Than Traditional RPCs**: Instead of waiting on the public Ethereum mempool, transactions sent through MEV Blocker get forwarded directly to all major builders, resulting in a faster inclusion rate. 
 
-## Why Integrate MEV Blocker?
+* **Multiple Endpoints**: MEV Blocker offers three different RPC endpoints that users can choose from based on their individual needs.
```
Comment (cowmarketing) [inline docs/mevblocker/README.mdx]: Theres more than 3 endpoints. Currently we have 5

File: docs/mevblocker/builders/fees/subscription-fees.md [hunk 3]
```diff
@@ -22,10 +22,36 @@ Despite the fact that the fee is per block, it will not be paid with each block:
...
 
 Finally, for builders who won fewer than 1% of blocks during a given week, the fee will be adjusted so that these builders pay as if they won exactly 1% of blocks during the week. In practice, at the end of each week, these builders pay:
 
 $$ 1\% (total\ blocks\ mined\ during\ the\ previous\ week) * 20\% M_{t-1} $$
 
-The rationale for this adjustment is that connecting a builder to MEV blocker poses a risk because of the potential leaking of transactions to the public mempool, which should be evaluated against the benefit of faster inclusion of MEV blocker transactions. We believe the risk outweighs the benefit for builders who win less than 1% of blocks. Although we do not want to penalize builders who may temporarily fall below this threshold too harshly, we also want to discourage builders that pose risk to the MEV Blocker service, from connecting in the first place.
```
Comment (cowmarketing) [inline docs/mevblocker/builders/fees/subscription-fees.md:31]: @gogonimago are we sure we want to have all this in a single page? I feel like it just makes the page super long

File: docs/mevblocker/concepts/mev-concepts/what-is-frontrunning.md [hunk 4]
```diff
@@ -1,66 +1,61 @@
...
 
-The frontrunning attack triggered an increase in the price of \$BOB tokens due to the additional demand created before the user's transaction was processed. When the user's original order of 145M \$BOB was executed, it was conducted at this now-inflated price, leading to user 0x3a7822 receiving way fewer \$BOB tokens, hence losing out on the value they could have otherwise captured.
+The frontrunning attack triggered an increase in the price of \$BOB tokens due to the additional demand created before the user's transaction was processed. When the user's original order of 145M \$BOB was executed, it was conducted at this now-inflated price, leading to user 0x3a7822 receiving significantly fewer \$BOB tokens, hence losing out on the value they could have otherwise captured.
 
-Jaredfromsubway.eth completed its MEV attack by selling its \$BOB tokens at an increased price, resulting in a profit, a second move that made this MEV attack a sandwich attack. In this scenario, the price slippage triggered by the frontrunning bot meant that user 0x3a7822 encountered a worse trade-off, ultimately making a loss while the MEV bot profited - an estimated gain of 0.1 ETH.
-
```
Comment (cowmarketing) [inline docs/mevblocker/concepts/mev-concepts/what-is-frontrunning.md:44]: ### An alternative frontrunning scenario

If user, 0x3a7822 identified a liquidation opportunity faster than anyone else and sent its transaction to the mempool to get the discounted assets from the liquidation in the first place, then Jaredfromsubway.eth would have seen this and would have replaced the users transactions with the transaction of its own, but with a higher gas fee, and thus more has an increased likelihood to of being processed faster, and effectively stealing the opportunity of capturing value away from the user.

why are we deleting this section @gogonimago ?

File: docs/mevblocker/concepts/order-flow-auction/order-flow-auction.md [hunk 5]
```diff
@@ -1,52 +1,97 @@
...
 
-MEV Blocker's Transaction Flow can be broken down into 6 major steps. In the picture below you can find a general overview of the flow of a transaction.
+MEV Blocker's transaction flow can be broken down into 6 main steps. The image below shows a general overview of the flow of a transaction.
 
 ![The Best Protection Under the Sun](/img/mevblocker/mevblocker_ofa.png)
 
-## User submits the transactions
+## 1. A user submits a transaction  
 
-This is the first step, where the user creates the transaction:
```
Comment (cowmarketing) [inline docs/mevblocker/concepts/order-flow-auction/order-flow-auction.md:13]: If we have a title, shouldnt this be followed by a sentence? no matter how straight forward?

File: docs/mevblocker/builders/fees/subscription-fees.md (hunk 3 above)
Comment (gogonimago) [reply docs/mevblocker/builders/fees/subscription-fees.md:31]: I think "Why do we charge a fee" shouldn't be its own page and should just be included in the top-line info on the fee itself. It's longer but it's not unbearable IMO

File: docs/mevblocker/concepts/mev-concepts/what-is-frontrunning.md (hunk 4 above)
Comment (gogonimago) [reply docs/mevblocker/concepts/mev-concepts/what-is-frontrunning.md:44]: I don't think there's a need to provide two different frontrunning scenarios. These are docs, not educational guides and if people want to learn more they can read our articles about them. In fact I thought about removing the three MEV explainers entirely from the docs and just sending people to our articles about them.

File: docs/mevblocker/concepts/order-flow-auction/order-flow-auction.md (hunk 5 above)
Comment (gogonimago) [reply docs/mevblocker/concepts/order-flow-auction/order-flow-auction.md:13]: Personally I don't think so, but it's a matter of style choice either way...

**Review by cowmarketing — APPROVED** [review]

**Review by marshymarsh — APPROVED** [review]

**Pull request merged**